package.json

```diff
@@ -24,7 +24,7 @@
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-react": "^7.12.5",
     "@babel/preset-typescript": "^7.12.1",
-    "@centreon/frontend-core": "github:centreon/frontend-core",
+    "@centreon/frontend-core": "github:centreon/frontend-core#d68881429f0a74ffd869aeaf1e2df10d0c4d00ea",
```

Comment:
Until https://github.com/centreon/centreon/pull/9355 is merged

Reply:
it's merged :)